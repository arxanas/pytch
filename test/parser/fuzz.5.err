EXPECTED_END_OF_ARGUMENT_LIST[1018] in test/parser/fuzz.5.pytch, line 1, character 21:
Error: I was expecting a ',' or ')' after the previous argument.
   +---------------------------------------------------------------------+
   | test/parser/fuzz.5.pytch                                            |
 1 | let foo = letprint(1let)                                            |
   |                     ^ Error: I was expecting a ',' or ')' after the |
   | previous argument.                                                  |
   +---------------------------------------------------------------------+
