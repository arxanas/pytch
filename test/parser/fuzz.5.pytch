let foo = letprint(1let)
