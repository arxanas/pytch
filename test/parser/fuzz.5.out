SyntaxTree
    LetExpr
        Token 'let'
        VariablePattern
            Leading ' '
            Token 'foo'
        <missing>
        Leading ' '
        Token '='
        FunctionCallExpr
            IdentifierExpr
                Leading ' '
                Token 'letprint'
            ArgumentList
                Token '('
                Argument
                    IntLiteralExpr
                        Token '1'
                    <missing>
                Leading 'let'
                Leading ''
                Token ')'
                Trailing '\n'
        Token DUMMY_IN ''
        <missing>
    Token EOF ''
